software/script/chameleon_cstruct.py

```diff
@@ -20,9 +20,11 @@ def parse_14a_scan_tag_result(data: bytearray):
     """
     return {
         'uid_size': data[10],
-        'uid_hex': data[0:data[10]].hex(),
+        'uid_hex': data[0:data[10]].hex(' '),
         'sak_hex': hex(data[12]).lstrip('0x').rjust(2, '0'),
-        'atqa_hex': data[13:15].hex().upper()
```

Comment:
removed `.upper()` because it's called anyways by the caller, added space separator for clarity